.github/actions/preview/message/update/action.yaml

```diff
@@ -11,6 +11,9 @@ inputs:
   preview-url:
     description: The preview url to add in the comment.
     required: true
+  issue-name:
+    description: The PR title to check if it involves an 'icons update'.
+    required: true
```

Comment:
This won't work, if you do not add the `issue-name` property in all of our workflows, who use this action.